### DISCUSSION TITLE
Kaoto Data Mapper timeframe question

### DISCUSSION THREAD

**Discussion by mjtrac** [KaotoIO/kaoto#1857] [Q&A] [answered]
Hi,

Kaoto, and its data mapper, look like fantastic tools.  However, when I try configuring an input with an xsd schema (FHIR) that uses <xs:include.../> to include other schemas, I get an error message indicating that including external items isn't implemented yet.  Does anyone know if/when this might be implemented, or can anyone offer a workaround?  Thanks in advance, and thanks for the beautiful product.

**Comment by lhein** [comment, marked as answer]
Thank you for the feedback. The feature you mentioned is on our radar and you can track the progress on issue #1816

**Reply by mjtrac** [reply]
Thanks.
